packages/vitest/src/node/config.ts

```diff
@@ -140,8 +140,8 @@ export function resolveConfig(
   if (resolved.minWorkers)
     resolved.minWorkers = Number(resolved.minWorkers)
 
-  // run benchmark sequentially by default
-  resolved.fileParallelism ??= mode !== 'benchmark'
+  // browser pool and benchmark are sequential by default
+  resolved.fileParallelism ??= resolved.pool !== 'browser' && mode !== 'benchmark'
```

Comment:
`resolved.pool` is always `threads` here, so this isn't right...

Reply:
It's also possible to override pool with `poolMatchGlobs`. 

The easiest solution here would be to set `resolved.browser.fileParallelism` before we override `resolved.fileParallelism`

Reply:
Thanks, that sounds good. Updated in https://github.com/vitest-dev/vitest/pull/5528/commits/ee772d137497399233d2c0a2935a92602d446614